pkg/steps/project_image.go

```diff
@@ -31,7 +31,14 @@ func (s *projectDirectoryImageBuildStep) Inputs() (api.InputDefinition, error) {
 	return nil, nil
 }
 
-func (*projectDirectoryImageBuildStep) Validate() error { return nil }
+func (s *projectDirectoryImageBuildStep) Validate() error {
+	for i, e := range s.config.BuildArgs {
+		if e.ValueFrom != nil {
+			fmt.Errorf("build_args[%d] not implemented: valueFrom", i)
```

Comment:
```suggestion
			fmt.Errorf("build_args[%d]: valueFrom must be set", i)
```

Is the implication that `Value` must be unset? If so, we should validate that. Do you have a reference for this?

Reply:
We support only `value` for now.

Confirmed by @vrutkovs 

https://coreos.slack.com/archives/CBN38N3MW/p1619182185023200?thread_ts=1619083815.472400&cid=CBN38N3MW

Reply:
```
oc explain build.spec.strategy.dockerStrategy.buildArgs
KIND:     Build
VERSION:  build.openshift.io/v1

RESOURCE: buildArgs <[]Object>

DESCRIPTION:
     buildArgs contains build arguments that will be resolved in the Dockerfile.
     See https://docs.docker.com/engine/reference/builder/#/arg for more
     details.

     EnvVar represents an environment variable present in a Container.

FIELDS:
   name	<string> -required-
     Name of the environment variable. Must be a C_IDENTIFIER.

   value	<string>
     Variable references $(VAR_NAME) are expanded using the previous defined
     environment variables in the container and any service environment
     variables. If a variable cannot be resolved, the reference in the input
     string will be unchanged. The $(VAR_NAME) syntax can be escaped with a
     double $$, ie: $$(VAR_NAME). Escaped references will never be expanded,
     regardless of whether the variable exists or not. Defaults to "".

   valueFrom	<Object>
     Source for the environment variable's value. Cannot be used if value is not
     empty.
```